src/components/button/styles/SButton.tsx

```diff
@@ -12,6 +12,14 @@ const getSize = (size?: ISizesWithDefault) => {
         padding: 0 5px;
```

Comment:
@nicolasshiken please remove all these changes that override fixes implemented in other PR.